npm/vue/src/index.ts

```diff
@@ -305,6 +307,8 @@ function failTestOnVueError (err, vm, info) {
   window.top.onerror(err)
 }
 
+enableAutoDestroy(beforeEach)
```

Comment:
This is what I was thinking - although based on my new understanding we should be using cypress events. I think this is okay for now!